test/go-report-card-test/Dockerfile

```diff
@@ -12,4 +12,4 @@ RUN go get github.com/gojp/goreportcard/cmd/goreportcard-cli
 
 RUN go get -u golang.org/x/tools/cmd/goimports
 
-CMD $GOPATH/bin/goreportcard-cli -v -t 90
\ No newline at end of file
+CMD $GOPATH/bin/goreportcard-cli -v -t 90
```

Comment:
does this threshold not mean anything/can it be removed? It's set to 93 in the script itself and it seems to adhere to that than this param

Reply:
it's the default threshold if you don't specify any command when calling docker run.